core/server/services/url/configs/v2.js

```diff
@@ -97,7 +97,7 @@ module.exports = [
         },
         events: {
             add: 'tag.added',
-            update: 'tag.edited',
+            update: ['tag.edited', 'tag.attached', 'tag.detached'],
```

Comment:
New events as we want to differentiate between regular edit and model attachment change